src/logic/exceptions/registry.ts

```diff
@@ -30,9 +30,6 @@ enum ErrorCodes {
   _701 = '701: Failed to save a localStorage item',
   _702 = '702: Failed to remove a localStorage item',
   _703 = '703: Error migrating localStorage',
-  _704 = '704: Failed to load a sessionStorage item',
```

Comment:
Why have you removed these? The `/load` and `/open` routes use the `sessionStorage` to retrieve the current `chainId`.

Reply:
I know but there's little value in having two sets of error messages depending on which storage this is. Besides, we're not even logging them. I can change `localStorage` -> `storage`.

Reply:
Alright, I would suggest generalising 701-703 then.